app/v2/notifications/post_notifications.py

```diff
@@ -63,6 +64,9 @@
 )
 
 
+# FIXME: POST requests to /v2/notifications/letter match this route and the
+# route hooked to the post_notification function below.  Proper execution
+# seems to depend on the order of the code (top-to-bottom route matching).
```

Comment:
is there a ticket for this FIXME? If so, can the # be added to this comment? If not, can the ticket be created and the # added to this comment?

Reply:
We should probably just make a ticket to go through and check the FIXME and TODO comments and make / check for tickets related to them.

Reply:
Ticket created: https://github.com/department-of-veterans-affairs/notification-api/issues/840